corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -92,14 +52,54 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function () {
                 self.showPaginationSpinner(false);
             }
         });
-
         return self;
     };
 
     var $releaseLogsTab = $('#release-logs-tab');
+    var appReleaseLogs = appReleaseLogsModel();
     if ($releaseLogsTab.length) {
-        var appReleaseLogModel = appReleaseLogsModel();
-        $releaseLogsTab.koApplyBindings(appReleaseLogModel);
+        $releaseLogsTab.koApplyBindings(appReleaseLogs);
+    }
+
+    // Main releases content
+    var releasesMainModel = hqImport('app_manager/js/releases/releases').releasesMainModel;
+    var o = {
+        currentAppVersion: initial_page_data('app_version') || -1,
+        recipient_contacts: initial_page_data('sms_contacts'),
+        download_modal_id: '#download-zip-modal',
+        latestReleasedVersion: initial_page_data('latestReleasedVersion'),
+        upstreamBriefs: initial_page_data('upstream_briefs'),
+        upstreamUrl: initial_page_data('upstream_url'),
+        appReleaseLogs,
```

Comment:
The test failure looks related.
I think its because this needs to be
`appReleaseLogs: appReleaseLogs`

I am surprised this is working?
Also, good to test this for a domain that does not have the feature flag enabled.

Reply:
`appReleaseLogs` is the shorthand for `appReleaseLogs: appReleaseLogs`, the tests might not be able to recognise it, will change that and check the tests again. 
Thanks for pointing that out, will test this again for a domain that does not have this feature flag.